src/common/acl_util.c

```diff
@@ -918,3 +918,79 @@ daos_acl_to_strs(struct daos_acl *acl, char ***ace_strs, size_t *ace_nr)
 
 	return 0;
 }
+
+#define D_FPRINTF(...)							\
```

Comment:
this wants to be in gurt header next to D_PRINTF

Reply:
Couldn't find a D_PRINTF, but I moved it into the same header where D_ASPRINTF lives.